lc0/src/mcts/search.cc

```diff
@@ -216,6 +219,9 @@ void Search::Worker() {
         for (Node* n : node->Children()) {
           float p = computation.GetPVal(idx_in_computation,
                                         n->GetMove().as_nn_index());
+          if(KPolicySoftmaxTemp != 1.0f && p != 1.0f && p > 0.0f){
```

Comment:
Oops, sorry, indeed I meant `KPolicySoftmaxTemp != 1.0f`, `p != 1.0f` is not really needed.
Unless that's the only move, it cannot realistically be 1.0. Same probably with 0.0, but that's not that clear.

Reply:
Ok, removed both the 1 and 0 checks.